packages/visx-wordcloud/src/Wordcloud.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import d3Cloud from 'd3-cloud';
+import { Group } from '@visx/group';
+import { BaseDatum, WordcloudConfig } from './types';
+import useWordcloud from './useWordcloud';
+
+export interface WordcloudProps<Datum extends BaseDatum> extends WordcloudConfig<Datum> {
+  children: (words: d3Cloud.Word[]) => React.ReactNode;
+}
+
+export default function Wordcloud<Datum extends BaseDatum>(props: WordcloudProps<Datum>) {
+  const { children, ...wordcloudConfig } = props;
+  const { width, height } = wordcloudConfig;
+  const words = useWordcloud(wordcloudConfig);
+
+  if (width === 0 || height === 0) return null;
+
+  return (
+    <svg width={wordcloudConfig.width} height={wordcloudConfig.height}>
```

Comment:
super minor but can update to `width`/`height` since you've already pulled them out

```suggestion
    <svg width={width} height=.height}>
```